README.md

```diff
@@ -1,72 +1,151 @@
-## Discovery
+<h1 align="center">NYPL Discovery Applicaiton</h1>
```

Comment:
Also - isn't necessarily something to fix now, but - it looks like some of these badges need to be updated. We don't use the `master` branch, so prob shouldn't include a badge for the `master` build status. The `development` and `production` badges would be:

```
[![Development Build Status](https://app.travis-ci.com/NYPL/discovery-front-end.svg?branch=development)](https://app.travis-ci.com/NYPL/discovery-front-end)
[![Production Build Status](https://app.travis-ci.com/NYPL/discovery-front-end.svg?branch=production)](https://app.travis-ci.com/NYPL/discovery-front-end)
```

The `<img>` equivalents don't appear to have anchor tags, so it seems github.com renders links to the image themselves, which feels less useful than linking to Travis. I'm not personally committed to keeping badges at all, so I wouldn't oppose abandoning them if you prefer

Reply:
> There a reason we can't just use markdown here to improve readability in plaintext readers?

Yes, because Markdown doesn't support center text. Using html tags, since most markdown parser support html, we can take advantage of some styling. Some processors do support markdown as `<center>text</center>` to center text but it still uses html tags syntax. 

Also, html is something most (if not all) frontend devs would understand.

Reply:
> We don't use the master branch, so prob shouldn't include a badge for the master build status. The development and production badges would be

Ah, perhaps we can go through the 211 branches in this repo and remove the ones which are stale. I was not aware (or forgot) we had a `production` branch.

I can point the master to production. This shouldn't be an issue.

Reply:
> The <img> equivalents don't appear to have anchor tags, so it seems github.com renders links to the image themselves

We can add anchors

Reply:
If we just want centering, couldn't we just use html for the div alone, e.g.
```
<div align="center">

# Discovery
...
[![Development Build Status](https://app.travis-ci.com/NYPL/discovery-front-end.svg?branch=development)](https://app.travis-ci.com/NYPL/discovery-front-end)
...
... other markdown stuff
</div>
```

Reply:
Is this something you'd like changed or a suggestion? Seems silly to change something that works for something else that works. Unless you'd prefer a minimum of Mark up in favor of Mark down.

Reply:
Sorry I haven't made my thinking clear. Yes I think we should keep html out of our README unless there's not another way to get the same rendered effect. The reason is that our READMEs are frequently viewed in plaintext editors by Developers looking for specific information. Although you're right that anyone viewing these files in a plaintext editor _understands_ HTML, it's not a format that favors quick digest. In general I think we should seek to reduce cognitive load in documentation. Character data that only serves formatting should be kept to a minimum because our brain has to skip over all of that to get to the actual information. This is an interesting area and I may be overlooking something, so happy to keep chatting about it if you feel strongly about it.

Reply:
Not at all. Simple changes. I've made them for you to review. I'm marking this chat as resolved and for you to re-review. Thanks